cloudflare-worker.js

```diff
@@ -32,21 +34,10 @@ async function handleRequest(request) {
   // 在 Cloudflare 中，HEAD 和 GET 请求带 body 会报错
   if (['HEAD', 'GET'].includes(request.method)) delete payload.body;
 
-  // 入参中如果包含了 stream=true，则表现形式为非流式输出
   const response = await fetch(fetchAPI, payload);
-  if (body && body.stream && body.stream === false) {
-    const results = await response.json();
-    return new Response(JSON.stringify(results), {
-      status: response.status,
-      headers: {
-        "Content-Type": "application/json",
-      },
-    });
-  } else {
-    return new Response(response.body, {
-      status: response.status,
-      statusText: response.statusText,
-      headers: response.headers,
-    });
-  }
-}
+  return new Response(response.body, {
```

Comment:
这么干，好像确实可以，是否走 stream 让 openai 来决定，我来测试下。

Reply:
> 完全由调用者来控制
 
You just need to forward the `body` as is. If the caller sets `stream: true`, it will be passed to OpenAI API.
There's no need for you to do any special handling for the streaming output of the OpenAI API.

<img width="1886" alt="image" src="https://user-images.githubusercontent.com/38807139/230654008-2cd31cac-4d50-49b4-bdf9-6f160407cc00.png">